connection.js

```diff
@@ -94,19 +94,22 @@ function connect(id, password, roomName) {
             JitsiConnectionEvents.CONNECTION_FAILED,
             connectionFailedHandler);
 
+        /* eslint-disable max-params */
         /**
          *
          */
-        function connectionFailedHandler(error, message, credentials) {
+        function connectionFailedHandler(error, message, credentials, details) {
```

Comment:
1. Details are always guaranteed to be passed in?
2. Nit: do you think it makes a difference is the esling-disabling were scoped to the function signature only instead of the entire function declaration?